tests/lib/Server/Input/Parser/AbstractDestinationLocationInputTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Rest\Server\Input\Parser;
+
+use Ibexa\Contracts\Core\Repository\LocationService;
+use Ibexa\Core\Repository\Values\Content\Location;
+use Ibexa\Rest\Server\Exceptions\ValidationFailedException;
+use Ibexa\Rest\Server\Input\Parser\AbstractDestinationLocationParser;
+use PHPUnit\Framework\MockObject\MockObject;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+abstract class AbstractDestinationLocationInputTest extends BaseTest
+{
+    private const int TESTED_LOCATION_ID = 22;
+
+    protected MockObject&LocationService $locationService;
```

Comment:
This looks okay-ish, but nut sure if we already agreed upon such code style rule.

Reply:
I think it is the only variation that PHPStan is accepting 😌